Canvas: Add element snapping and alignment (#80407)
grafana/grafana: docs/sources/panels-visualizations/visualizations/canvas/index.md

{{< figure src="/static/img/docs/canvas-panel/canvas-context-menu-9-2-0.png" max-width="750px" caption="Canvas element context menu" >}}

### Element snapping and alignment

> When moving elements around the canvas, snapping and alignment guides to help you create more precise layouts.

Reviewer: ```suggestion
When you're moving elements around the canvas, snapping and alignment guides help you create more precise layouts.
```

> 

Author: should we add a note here that this feature only works when the panel is not zoomed in at all? (i.e. default scale) cc @drew08t

When you're moving elements around the canvas, snapping and alignment guides help you create more precise layouts.

{{% admonition type="note" %}}

> For the time being, element snapping and alignment only works when the canvas is not zoomed in.

Reviewer: ```suggestion
Currently, element snapping and alignment only works when the canvas is not zoomed in.
```
